bus/routes.go

```diff
@@ -583,23 +583,43 @@ func (b *Bus) hostsPubkeyHandlerGET(jc jape.Context) {
 	}
 }
 
-func (b *Bus) hostsResetLostSectorsPOST(jc jape.Context) {
-	var hostKey types.PublicKey
-	if jc.DecodeParam("hostkey", &hostKey) != nil {
+func (b *Bus) hostsScanHandlerPOST(jc jape.Context) {
+	ctx := jc.Request.Context()
+
+	// decode the request
+	var rsr api.HostScanRequest
+	if jc.Decode(&rsr) != nil {
 		return
 	}
-	err := b.store.ResetLostSectors(jc.Request.Context(), hostKey)
-	if jc.Check("couldn't reset lost sectors", err) != nil {
+
+	// only scan hosts if we are online
+	if len(b.s.Peers()) == 0 {
+		jc.Error(errors.New("not connected to the internet"), http.StatusServiceUnavailable)
 		return
 	}
+
+	// scan host
+	var errStr string
+	settings, priceTable, elapsed, err := b.scanHost(ctx, time.Duration(rsr.Timeout), rsr.HostKey, rsr.HostIP)
+	if err != nil {
+		errStr = err.Error()
+	}
+
+	jc.Encode(api.HostScanResponse{
+		Ping:       api.DurationMS(elapsed),
+		PriceTable: priceTable,
+		ScanError:  errStr,
```

Comment:
I think the error should be returned as a `ScanError` iff it's an error that occurred when actually attempting to scan the host. Currently it will also include the error returned from `RecordHostScans`.